README.md

```diff
@@ -18,13 +18,98 @@ This app provides a filtered dashboard for the existing Falcon console Detection
 
 You can use this app to configure automated and manual actions for specific tactics and techniques.
 
-## Basic information
+## Prerequisites
 
-### Dependencies
+* The Foundry CLI (instructions below).
+* Go v1.21+ (needed if modifying the app's functions). See https://go.dev/learn for installation instructions.
+* Yarn (needed if modifying the app's UI). See https://yarnpkg.com/getting-started for installation instructions.
 
-- Foundry CLI
-- Go v1.21+ (needed if modifying functions). See https://go.dev/learn/ for instructions to install.
-- YARN (needed if modifying UI). See https://yarnpkg.com/getting-started for instructions to install.
+### Install the Foundry CLI
+
+You can install the Foundry CLI with Scoop on Windows or Homebrew on Linux/macOS.
+
+**Windows**:
+
+Install [Scoop](https://scoop.sh/). Then, add the Foundry CLI bucket and install the Foundry CLI.
+
+```shell
+scoop bucket add foundry https://github.com/crowdstrike/scoop-foundry-cli.git
+scoop install foundry
+```
+
+Or, you can download the [latest Windows zip file](https://assets.foundry.crowdstrike.com/cli/latest/foundry_Windows_x86_64.zip), expand it, and add the install directory to your PATH environment variable.
+
+**Linux and macOS**:
+
+Install [Homebrew](https://docs.brew.sh/Installation). Then, add the Foundry CLI repository to the list of formulae that Homebrew uses and install the CLI:
+
+```shell
+brew tap crowdstrike/foundry-cli
+brew install foundry
+```
+
+Run `foundry version` to verify it's installed correctly.
+
+## Getting Started
+
+Clone this sample to your local system, or [download as a zip file](https://github.com/CrowdStrike/foundry-sample-mitre/archive/refs/heads/main.zip).
+
+```shell
+git clone https://github.com/CrowdStrike/foundry-sample-mitre
+cd foundry-sample-mitre
+```
+
+Log in to Foundry:
+
+```shell
+foundry login
+```
+
+Select the following permissions:
+
+- [x] Run RTR Scripts
+- [x] Run, execute, and test Workflows
+- [x] Run, execute, and test API integrations
+- [x] Run, execute, and test LogScale queries
+- [ ] (optional) Generate mock data to test your app
```

Comment:
I'm not sure if this the correct set of permissions this app needs. Can I find out by reading the `manifest.yml`?